pyvista/core/utilities/geometric_sources.py

```diff
@@ -3384,3 +3386,192 @@ def _make_axes_parts(
+    def resolution(self) -> tuple[int, int, int]:  # numpydoc ignore=RT01
+        """Return or set the resolution of the planes."""
+        return self._resolution
+
+    @resolution.setter
+    def resolution(self, resolution: int | VectorLike[int]):  # numpydoc ignore=GL08
+        self._resolution = _validation.validate_array3(
+            resolution, broadcast=True, to_tuple=True, name='resolution'
+        )
+
+    @property
+    def bounds(self) -> BoundsLike:  # numpydoc ignore=RT01
+        """Return or set the bounds of the planes."""
+        return self._bounds
+
+    @bounds.setter
+    def bounds(self, bounds: BoundsLike):  # numpydoc ignore=GL08
+        self._bounds = _validation.validate_array(
+            bounds, dtype_out=float, must_have_length=6, to_tuple=True, name='bounds'
+        )
+
+    @property
+    def names(self) -> tuple[str, str, str]:  # numpydoc ignore=RT01
+        """Return or set the names of the planes."""
+        return self._names
+
+    @names.setter
+    def names(self, names: Sequence[str]):  # numpydoc ignore=GL08
+        _validation.check_instance(names, (tuple, list), name='names')
+        _validation.check_iterable_items(names, str, name='names')
+        _validation.check_length(names, exact_length=3, name='names')
+        self._names = cast(Tuple[str, str, str], tuple(names))
+
+    def update(self):
+        """Update the output of the source."""
+        ORIGIN = (0.0, 0.0, 0.0)
+
+        # Unpack vars
+        xy_source, yz_source, zx_source = self._plane_sources
+        x_res, y_res, z_res = self.resolution
+        x_min, x_max, y_min, y_max, z_min, z_max = self.bounds
+
+        # Compute bounds-related vars
+        x_size, y_size, z_size = x_max - x_min, y_max - y_min, z_max - z_min
+        center = (x_max + x_min) / 2, (y_max + y_min) / 2, (z_max + z_min) / 2
+
+        # Update plane sources
+        xy_source.SetXResolution(x_res)
+        xy_source.SetYResolution(y_res)
+        xy_source.SetPoint1(x_size, 0.0, 0.0)
+        xy_source.SetPoint2(0.0, y_size, 0.0)
+        xy_source.SetOrigin(ORIGIN)
+        xy_source.SetCenter(center)
+        xy_source.Update()
+
+        yz_source.SetXResolution(y_res)
+        yz_source.SetYResolution(z_res)
+        yz_source.SetPoint1(0.0, y_size, 0.0)
+        yz_source.SetPoint2(0.0, 0.0, z_size)
+        yz_source.SetOrigin(ORIGIN)
+        yz_source.SetCenter(center)
+        yz_source.Update()
+
+        zx_source.SetXResolution(z_res)
+        zx_source.SetYResolution(x_res)
+        zx_source.SetPoint1(0.0, 0.0, z_size)
+        zx_source.SetPoint2(x_size, 0.0, 0.0)
+        zx_source.SetOrigin(ORIGIN)
+        zx_source.SetCenter(center)
+        zx_source.Update()
+
+        # Update the output
+        output = self._output
+        for index, name, source, plane, sign in zip(
+            range(3), self.names, self._plane_sources, output, self.normal_sign
+        ):
+            plane.copy_from(source.GetOutput())
```

Comment:
Related to https://github.com/pyvista/pyvista/pull/6366/files#r1676027612

We can remove this line if we also modify `pv.PlaneSource` to make use of `copy_from` mechanics instead of `pv.wrap` in the output.

Most of the pyvista sources seem to use `pv.wrap` in the output. But updating the output this way creates a new output instance every time. This differs from vtk sources, where the output always refers to the same instance.
e.g.
``` python
import pyvista as pv
import vtk as vtk
vtk_source = vtk.vtkPlaneSource()
assert vtk_source.GetOutput() is vtk_source.GetOutput() # success
pv_source = pv.PlaneSource()
assert pv_source.output is pv_source.output  # fail
```

This is an issue because this class  (`OrthogonalPlanesSource`)  stores the outputs in `__init__`, and any updates thereafter therefore needs to only update the outputs without creating new instances.